[2.9] Misc Learn updates (#3574)
astronomer/docs: learn/airflow-components.md

title: "Airflow components"
sidebar_label: "Core components"
id: airflow-components

> description: "Understand the core components of Apache Airflow 2.0+. Review their functions and find out which components to run for specific use cases."

Author: @kentdanas You said something about a 2.2 reference... that was not the oldest one 😅

    - Your tasks require very specific resource configurations.
    - Your tasks run infrequently, and you don't want to incur worker resource costs when they aren't running.

> - **[CeleryKubernetes Executor](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/executor/celery_kubernetes.html)**: Allows you to run both, a CeleryExecutor and a KubernetesExecutor in the same Airflow instance. Which executor a specific task uses is determined using the `queue` parameter. You need to install both, the [Celery](https://registry.astronomer.io/providers/apache-airflow-providers-celery/versions/latest) and [CNCF Kubernetes](https://registry.astronomer.io/providers/apache-airflow-providers-cncf-kubernetes/versions/latest) providers to use this executor.

Reviewer: ```suggestion
- **[CeleryKubernetes Executor](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/executor/celery_kubernetes.html)**: Allows you to run both a CeleryExecutor and a KubernetesExecutor in the same Airflow instance. The `queue` parameter determines which executor that a specific task uses. You need to install both, the [Celery](https://registry.astronomer.io/providers/apache-airflow-providers-celery/versions/latest) and [CNCF Kubernetes](https://registry.astronomer.io/providers/apache-airflow-providers-cncf-kubernetes/versions/latest) providers to use this executor.
```
